content/en/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,127 @@
+---
+title: About cgroup v2
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+On Linux, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+constrain resources that are allocated to processes.
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and the
+underlying container runtime need to interface with cgroups to enforce
+[resource mangement for pods and
+containers](/docs/concepts/configuration/manage-resources-containers/) which
```

Comment:
It is always desirable to keep links on the same line. e.g.

```suggestion
[resource mangement for pods and containers](/docs/concepts/configuration/manage-resources-containers/) which
```

Reply:
Done